virtool/utils.py

```diff
@@ -30,7 +31,7 @@
 ]
 
 
-def average_list(list1, list2):
+def average_list(list1: list, list2: list) -> list:
```

Comment:
Use `numbers.Number` to encompass all Python number types.

```suggestion
def average_list(list1: List[Number], list2: List[Number]) -> List[Number]:
```